build-aux/snap/apparmor/af_names.h

```diff
@@ -0,0 +1,223 @@
+#ifndef AF_UNSPEC
```

Comment:
This is generated when building AppArmor - but it gets generated against the libc headers which are available at build time. Since snapd is core16 it gets built against xenial libc which doesn't support a bunch of different address families so we can't generate it at build time otherwise AppArmor never knows about those address families. So instead I generated it on an impish machine and vendored it in.

Reply:
thanks for explaining, could we at least add a comment to the top of this file that it was auto-generated on impish as part of building apparmor there, etc. ?

Reply:
Good idea - done in 80277ed89e